packages/calypso-build/package.json

```diff
@@ -52,6 +55,7 @@
 		"terser-webpack-plugin": "1.2.3",
 		"thread-loader": "2.1.2",
 		"webpack": "4.29.6",
+		"webpack-cli": "^3.3.0",
```

Comment:
I prefer having this as `^` range (since this is a package) but note that everything else here is strict version pinned. Shall we open up other versions with `^`, too?

Reply:
I'll remove the `^` for now. Introducing it (or `~`) for all dependencies isn't a change we should bury in this PR but file a separate one for.